Yubico.YubiKey/src/Yubico/YubiKey/Piv/PivAlgorithmExtensions.cs

```diff
@@ -104,18 +258,23 @@ public static class PivAlgorithmExtensions
         /// <returns>
         /// An int, the size, in bits, of a key of the given algorithm.
         /// </returns>
-        public static int KeySizeBits(this PivAlgorithm algorithm) => algorithm switch
+        public static int KeySizeBits(this PivAlgorithm algorithm)
         {
-            PivAlgorithm.Rsa1024 => 1024,
-            PivAlgorithm.Rsa2048 => 2048,
-            PivAlgorithm.Rsa3072 => 3072,
-            PivAlgorithm.Rsa4096 => 4096,
-            PivAlgorithm.EccP256 => 256,
-            PivAlgorithm.EccP384 => 384,
-            PivAlgorithm.TripleDes => 192,
-            PivAlgorithm.Pin => 64,
-            _ => 0,
-        };
+            // Special handling for PIN algorithm
+            if (algorithm == PivAlgorithm.Pin)
+            {
+                return 64;
+            }
+
+            if (algorithm == PivAlgorithm.None)
+            {
+                return 0;
+            }
```

Comment:
See my comments about using `null!` [here](https://github.com/Yubico/Yubico.NET.SDK/pull/210/files#diff-26b2cd5dfdf8b648101a4bf51848f1c7e02b1b12bebd9a38ac312d8bd8478125R97). Having specific KeyDefinitions for .None and .Piv solve this special handling.